biome.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git",
+    "useIgnoreFile": true,
+    "defaultBranch": "main"
+  },
+  "files": { "ignoreUnknown": false, "ignore": [] },
+  "formatter": {
+    "enabled": true,
+    "useEditorconfig": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80,
+    "attributePosition": "auto",
+    "bracketSpacing": true,
+    "ignore": [
+      "./wwwroot/data/regionids/*",
+      "**/yarn.lock",
+      "**/*.scss.d.ts",
+      "./wwwroot/third_party/*",
+      "doc/acknowledgements/attributions.md",
+      "./wwwroot/test/init/sdmx-abs.json"
```

Comment:
This is a 1.7 MB file that hasn't been updated since 2017, so put it in the ignore list to not slow down the formatting that is done.